src/main/resources/io/fabric8/elasticsearch/plugin/kibana/applications_mapping.json

```diff
@@ -0,0 +1,6 @@
+{
+   "title":"$TITLE$",
+   "timeFieldName":"time",
```

Comment:
This is the value directly exported out of Kibana for this index.  The mapping document should be provided 'mostly' complete.  It's a field for applications and I don't  see the point in substituting it in when it can just be provided as needed by whom ever is providing the document. I will update the class to remove 'time_field_name'